TESTING.md

```diff
@@ -109,6 +109,59 @@ mvn clean install -pl janusgraph-es -Delasticsearch.docker.image=elasticsearch
 mvn clean install -pl janusgraph-es -Delasticsearch.docker.version=1.5.1 -Delasticsearch.docker.image=elasticsearch
 ```
 
+### Running CQL Tests
+
+**Note** Running CQL tests require Docker.
+
+CQL tests are executed using [testcontainers-java](https://www.testcontainers.org/). 
+CQL tests can be executed against a Cassandra 2 using the profile `cassandra2`, 
+a Cassandra 3 using the profile `cassandra3`, or a Scylla 3 using the profile `scylladb`.
+
+```bash
+mvn clean install -pl janusgraph-cql -Pcassandra2-murmur
+mvn clean install -pl janusgraph-cql -Pcassandra3-murmur
+mvn clean install -pl janusgraph-cql -Pscylladb
```

Comment:
Is `scylladb` profile tested somehow? Should we add the tests into `.travis.yml` for `[full build]?`

Reply:
I will add it to the issue.